app/bundles/LeadBundle/Form/Type/LeadFieldsType.php

```diff
@@ -44,6 +44,7 @@ public function setDefaultOptions(OptionsResolverInterface $resolver)
                 if ($options['with_tags']) {
                     $fieldList['Core']['tags'] = 'mautic.lead.field.tags';
                 }
+                $fieldList['Core']['notifications'] = 'mautic.lead.field.notifications';
```

Comment:
I think this code is ok If  it's part a Contact field condition.
We should create new campain condition item similar like Contact owner https://github.com/mautic/mautic/pull/3980, for example Contact notification. But in next release